pages/contributionsList.Vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <h1>Placeholder for the contributions listing</h1>
+</template>
+
+<script>
+    export default {
+      name: 'Contributions-List',
+    }
+</script>
+
+<style scoped>
+
+</style>
```

Comment:
This file looks like a dup of `data_collector/ContributionsList.Vue`. Is it needed?

Reply:
Let's discuss this offline